File: benchmark/py/01_fibonacci.py
# フィボナッチ数計算 (再帰) --- 01_fibonacci.nako3 と同じアルゴリズム
import sys


def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)


sys.setrecursionlimit(10000)
print("fib(30) = {}".format(fib(30)))

File: benchmark/js/01_fibonacci.js
// フィボナッチ数計算 (再帰) --- 01_fibonacci.nako3 と同じアルゴリズム
function fib (n) {
  if (n <= 1) { return n }
  return fib(n - 1) + fib(n - 2)
}

console.log(`fib(30) = ${fib(30)}`)
